app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('miq.charts').directive('heatMap', ['$timeout',
+  function($timeout) {
```

Comment:
There are no changes to patternfly 2.6 that are required by angular-patternfly 2.6.